modules/contact/test/tests/bouncing-block-contact/tests

```diff
@@ -184,4 +184,9 @@
       allow_test_objects = True
     []
   []
+  [bouncing-block-ranfs-action]
```

Comment:
Rather than adding a new input file for ranfs, can't we just convert some of the existing tests to use the Contact action?

Reply:
Thanks Ben! All the comments have been addressed via this updated PR.